### DISCUSSION TITLE
Theming

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#578] [Feature requests]
Would be nice to be able to theme the admin panel.

**Comment by nhorton** [comment]
Is theming still expected in the near-term? Alternatively, are there any examples publicly of Tailwind CSS files where people have done it without the full support being released?

**Reply by adrianthedev** [reply]
We'll start to flesh out theming starting Q1 2024.

What are you expecting with this feature? What's level of customization are you expecting?

**Comment by adrianthedev** [comment]
I started to flesh out this feature [here](https://github.com/avo-hq/avo/issues/631). Closing this.
